packages/databases-collections/src/stores/rename-collection.ts

```diff
@@ -28,7 +56,27 @@ export function activateRenameCollectionPlugin(
   const onRenameCollection = (ns: { database: string; collection: string }) => {
     const collections: { name: string }[] =
       instance.databases.get(ns.database)?.collections ?? [];
-    store.dispatch(open(ns.database, ns.collection, collections));
+    loadSavedQueriesAndAggregations(`${ns.database}.${ns.collection}`)
+      .then((areSavedQueriesAndAggregationsImpacted) => {
+        store.dispatch(
+          open({
+            db: ns.database,
+            collection: ns.collection,
+            collections,
+            areSavedQueriesAndAggregationsImpacted,
+          })
+        );
+      })
+      .catch(() => {
```

Comment:
I can handle the defaulting in `checkIfSavedQueriesAndAggregationsExist`.  However, we'd still need a catch block because otherwise eslint complains that we have a promise without a catch on it - so I decided to use the catch block to open the rename collection plugin by if there are any errors.

would you prefer a different approach?

Reply:
I think if you move all the error handling logic to the method, it would be also safe to add `void` before the promise. Or we can keep the catch, but I'd not dispatch the open from there in that case because it would be unexpected at that point (but we might want to log the error)